microsoft-edge/webview2/index.md

```diff
@@ -167,6 +167,14 @@ You may run WebView2 apps on the following versions of Windows.
 > [!IMPORTANT]
 > \*\* WebView2 support for Windows 7 and Windows Server 2008 R2 has the same support cycle as Microsoft Edge.  For more information, navigate to [Microsoft Edge supported Operating Systems][DeployedgeMicrosoftEdgeSupportedOS].  
 
+WebView2 also works in the following Windows Embedded releases
+
+*   Windows IoT Enterprise LTSC x32
+*   Windows IoT Enterprise LTSC x64
+*   Windows IoT Enterprise 21h1 x64
```

Comment:
Can we replace with the product names?

Reply:
Those are the product names as far as I know.

Reply:
https://docs.microsoft.com/en-us/windows/iot/iot-enterprise/getting_started